src/test/lifecycle/common.js

```diff
@@ -44,6 +44,7 @@ exports.empty_filter_marker_lifecycle_configuration = empty_filter_marker_lifecy
 
 function date_lifecycle_configuration(Bucket, Key) {
     const now = new Date(Date.now());
+    now.setDate(now.getDate() + 1); // Increases date by 1 day; past/current dates can't be set
```

Comment:
Could you change the name of the variable? as it is not "now" anymore... (I saw it in more places).

Reply:
changed to "date"